docs/framework/configure-apps/file-schema/trace-debug/add-element-for-listeners-for-trace.md

```diff
@@ -78,7 +78,9 @@ Adds a listener to the **Listeners** collection.
 |<xref:System.Diagnostics.EventLogTraceListener?displayProperty=nameWithType>|The name of the name of an existing event log source.|  
 |<xref:System.Diagnostics.EventSchemaTraceListener?displayProperty=nameWithType>|The name of the file that the <xref:System.Diagnostics.EventSchemaTraceListener> writes to.|  
 |<xref:System.Diagnostics.TextWriterTraceListener?displayProperty=nameWithType>|The name of the file that the <xref:System.Diagnostics.TextWriterTraceListener> writes to.|  
-|<xref:System.Diagnostics.XmlWriterTraceListener?displayProperty=nameWithType>|The name of the file that the <xref:System.Diagnostics.XmlWriterTraceListener> writes to.|  
+|<xref:System.Diagnostics.XmlWriterTraceListener?displayProperty=nameWithType>|The name of the file that the <xref:System.Diagnostics.XmlWriterTraceListener> writes to.|
```

Comment:
There is another similar table here: <https://github.com/dotnet/docs/blob/0f1e282e3dd56eb2baeb5eb14d0b73903e983a43/docs/framework/configure-apps/file-schema/trace-debug/add-element-for-sharedlisteners.md?plain=1#L66-L75>